meta-balena-common/recipes-containers/docker-disk/docker-disk.inc

```diff
@@ -34,19 +34,22 @@ python () {
         bb.fatal("docker-disk: Docker needs to run on your host and current user must be able to use it.")
 }
 
+HOSTAPP_PLATFORM_armv5 = "linux/arm/v5"
+HOSTAPP_PLATFORM_armv6 = "linux/arm/v5"
+HOSTAPP_PLATFORM_armv7a = "linux/arm/v7"
+HOSTAPP_PLATFORM_armv7ve = "linux/arm/v7"
+HOSTAPP_PLATFORM_aarch64 = "linux/arm64"
+HOSTAPP_PLATFORM_x86 = "linux/i386"
+HOSTAPP_PLATFORM_x86-64 = "linux/amd64"
+HOSTAPP_PLATFORM_intel-quark = "linux/i386"
+
 HELLO_REPOSITORY_armv5 = "arm32v5/hello-world"
-HELLO_PLATFORM_armv5 = "linux/arm/v5"
 HELLO_REPOSITORY_armv6 = "arm32v5/hello-world"
-HELLO_PLATFORM_armv6 = "linux/arm/v5"
 HELLO_REPOSITORY_armv7a = "arm32v7/hello-world"
-HELLO_PLATFORM_armv7a = "linux/arm/v7"
 HELLO_REPOSITORY_armv7ve = "arm32v7/hello-world"
-HELLO_PLATFORM_armv7ve = "linux/arm/v7"
 HELLO_REPOSITORY_aarch64 = "arm64v8/hello-world"
-HELLO_PLATFORM_aarch64 = "linux/arm64"
 HELLO_REPOSITORY_x86 = "i386/hello-world"
-HELLO_PLATFORM_x86 = "linux/i386"
 HELLO_REPOSITORY_x86-64 = "amd64/hello-world"
-HELLO_PLATFORM_x86-64 = "linux/amd64"
 HELLO_REPOSITORY_intel-quark = "i386/hello-world"
-HELLO_PLATFORM_intel-quark = "linux/i386"
+
+HOSTEXT_IMAGES ?= "balena/default-hostext"
```

Comment:
Do we need a non-existing default? Can't we just leave this empty?

Reply:
Good point. It was easier for me to test with but it should be left empty as default.